features/createUser.feature

```diff
@@ -7,4 +7,5 @@ Feature: Create user
     Given I have a users name
         And I have a users id
     When I create a user
-    Then I receive a user object
+    Then I receive a HTTP 201 code
```

Comment:
I think maybe HTTP 201 code is too precise? Could it be "a succesful creation response"? Reads better from a functionality perspective.